yaksh/templates/yaksh/quizzes_user.html

```diff
@@ -7,6 +7,13 @@
             <center>{{ msg }}</center>
         </div>
     {% endif %}
+    {% if user.profile.is_moderator %}
+        <div class="row">
+            <a href="{{URL_ROOT}}/exam/toggle_moderator/" class="btn btn-primary pull-right" style="margin-top: 20px; margin-bottom: 20px;">
+              Switch To Moderator
```

Comment:
I feel the buttons to switch should be on the top nav bar. Always visible to the users.
If majority are fine with this then I am okay with it.

Reply:
In moderator templates, the nav bar becomes very very crowded because the button has 3 words.